<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Width</string>
           </property>
           <widget class="QSpinBox" name="spinBox">
            <property name="geometry">
             <rect>
              <x>1</x>
              <y>70</y>
              <width>241</width>
              <height>22</height>
             </rect>
            </property>
            <property name="maximum">
             <number>25</number>
            </property>
           </widget>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>170</y>
              <width>243</width>
              <height>181</height>
             </rect>
            </property>
            <property name="title">
             <string>GroupBox</string>
            </property>
            <widget class="QSlider" name="SliderRed">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>40</y>
               <width>221</width>
               <height>22</height>
              </rect>
             </property>
             <property name="maximum">
              <number>255</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
            <widget class="QSlider" name="SliderGreen">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>90</y>
               <width>221</width>
               <height>22</height>
              </rect>
             </property>
             <property name="maximum">
              <number>255</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
            <widget class="QSlider" name="SliderBlue">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>140</y>
               <width>221</width>
               <height>22</height>
              </rect>
             </property>
             <property name="maximum">
              <number>255</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
            <widget class="QLabel" name="label">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>20</y>
               <width>47</width>
               <height>13</height>
              </rect>
             </property>
             <property name="text">
              <string>Red</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_2">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>70</y>
               <width>47</width>
               <height>13</height>
              </rect>
             </property>
             <property name="text">
              <string>Green</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_3">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>120</y>
               <width>47</width>
               <height>13</height>
              </rect>
             </property>
             <property name="text">
              <string>Blue</string>
             </property>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Style</string>
           </property>
           <widget class="QComboBox" name="comboBox">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>60</y>
              <width>231</width>
              <height>22</height>
             </rect>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="Pattern" name="frame_4">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Pattern</class>
   <extends>QFrame</extends>
   <header>pattern.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
